src/ray/raylet/runtime_env_agent_client.cc

```diff
@@ -276,7 +274,7 @@ class HttpRuntimeEnvAgentClient : public RuntimeEnvAgentClient {
       : io_context_(io_context),
         session_pool_(session_pool_size),
         address_(address),
-        port_str_(std::to_string(port)),
```

Comment:
`std::to_string` is known to suffer hidden performance issue, with a global lock for timezone.

Copy from [doc](https://en.cppreference.com/w/cpp/string/basic_string/to_string):
```
std::to_string relies on the current C locale for formatting purposes, and therefore concurrent calls to std::to_string from multiple threads may result in partial serialization of calls.
```